pom.xml

```diff
@@ -76,6 +77,19 @@
         </dependency>
 
         <dependency>
+            <groupId>junit</groupId>
```

Comment:
This was needed due to ES checking for 'jar hell' when booting, even in test. Junit brings in hamcrest-core, which conflicts with the usage of hamcrest-all in the project's tests.